content/1_docs/2_cookbook/7_extensions/0_first-panel-view/recipe.txt

```diff
@@ -0,0 +1,358 @@
+Title: My first Panel view
+
+----
+
+Description: Create your first custom Panel view from scratch
+
+----
+
+Published: 2019-11-20
+
+----
+
+Text:
+
+For our very first Panel view, we will query a public API and display the results in a nice card layout.
+
+(screenshot: movie-reviews-custom-view.png)
+
+<info>
+For the purposes of this recipe, we assume that Parcel is already installed globally as described in the (link: docs/guide/plugins/plugin-setup-panel text: Panel plugin setup guide).
+
+You can install the (link: https://github.com/getkirby/pluginkit/tree/4-panel text: pluginkit) as a basis or create the file structure we need manually, that's up to you. Also, it doesn't matter if you use the PlainKit or the StarterKit as a starting point.
```

Comment:
According to the styleguide: Pluginkit, Plainkit, Starterkit

Also applies to other occurrences.